docs/releasing.md

```diff
@@ -31,71 +31,20 @@ We follow [semver](https://semver.org/) versioning semantics (`vMajor.Minor.Patc
 ## Release Process
 
 ```sh
-# use a custom lighthouse-pristine checkout to make sure your dev files aren't involved.
-
-# * Install the latest.*
-yarn
-
-# * Bump it *
-yarn version --no-git-tag-version
-# manually bump extension v in clients/extension/manifest.json
-yarn update:sample-json
-
-# * Build it. This also builds the cli, extension, and viewer. *
-yarn build-all
-
-# * Test err'thing *
-echo "Test the CLI."
-yarn start "https://example.com" --view
-yarn smoke
-
-echo "Test the extension"
-# ...
-
-echo "Test a fresh local install"
-# (starting from lighthouse-pristine root...)
-npm pack
-cd ..; rm -rf tmp; mkdir tmp; cd tmp
-npm init -y
-npm install ../lighthouse-pristine/lighthouse-*.tgz
-npm explore lighthouse -- npm run smoke
-npm explore lighthouse -- npm run chrome # try the manual launcher
-npm explore lighthouse -- npm run fast -- http://example.com
-cd ..; rm -rf ./tmp;
-
-cd ../lighthouse-pristine; command rm -f lighthouse-*.tgz
-
-echo "Test the lighthouse-viewer build"
-# Manual test for now:
-# Start a server in dist/viewer/ and open the page in a tab. You should see the viewer.
-# Drop in a results.json or paste an existing gist url (e.g. https://gist.github.com/ebidel/b9fd478b5f40bf5fab174439dc18f83a).
-# Check for errors!
-cd dist/viewer ; python -m SimpleHTTPServer
-# go to http://localhost:8000/
-
-# * Update changelog *
-git fetch --tags
-yarn changelog
-# add new contributors, e.g. from git shortlog -s -e -n v2.3.0..HEAD
-#    and https://github.com/GoogleChrome/lighthouse/graphs/contributors
-echo "Edit the changelog for readability and brevity"
-
-# * Put up the PR *
-echo "Branch and commit the version bump."
-git checkout -b bumpv240
-git commit -am "2.4.0"
-echo "Generate a PR and get it merged."
-
-echo "Once it's merged, pull master and tag the (squashed) commit"
-git tag -a v2.4.0 -m "v2.4.0"
-git push --tags
-
-
-# * Deploy-time *
-echo "Rebuild extension and viewer to get the latest, tagged master commit"
-yarn build-all;
-
-# zip the extension files
+# Run the tests
+bash ./lighthouse-core/scripts/publish-test.sh
+# Prepare the commit
+bash ./lighthouse-core/scripts/publish-prepare-commit.sh
+
+# Open the PR and await merge...
+echo "It's been merged! 🎉"
+
+# Run the tests again :)
+bash ./lighthouse-core/scripts/publish-test.sh
+# Package everything for publishing
+bash ./lighthouse-core/scripts/publish-prepare-package.sh
+
+# Upload the extension
 node build/build-extension.js package; cd dist/extension-package/
 echo "Go here: https://chrome.google.com/webstore/developer/edit/blipmdconlkpinefehnmjammfjpmpbjk "
```

Comment:
might as well open a browser tab for this.

```suggestion
open "https://chrome.google.com/webstore/developer/edit/blipmdconlkpinefehnmjammfjpmpbjk"
```

Reply:
heh, yeah :) 👍